app/models/sponsorship.rb

```diff
@@ -44,13 +44,16 @@ def validate_tag_uniqueness
     return unless self.class.where(sponsorable: sponsorable, level: :tag)
       .exists?(["expires_at > ? AND id != ?", Time.current, id.to_i])
 
-    errors.add(:level, "The tag is already sponsored")
+    errors.add(:level, I18n.t("models.sponsorship.already_sponsored"))
   end
 
   def validate_level_uniqueness
     return unless self.class.where(organization: organization)
       .exists?(["level IN (?) AND expires_at > ? AND id != ?", METAL_LEVELS, Time.current, id.to_i])
 
-    errors.add(:level, "You can have only one sponsorship of #{METAL_LEVELS.join(', ')}")
+    errors.add(:level,
+               I18n.t("models.sponsorship.only_one_level", levels: METAL_LEVELS.map do |l|
```

Comment:
Please move the `map` call out of here and assign it to a variable.

Reply:
Do you mean simply:
``` ruby
levels = METAL_LEVELS.map {...}.to_sentence(...)
errors.add(:level, I18n.t("...", levels: levels))
```

Reply:
Yes, that's what I meant, thanks!